cmd/terramate/main.go

```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
```

Comment:
why this comment style?

Reply:
I copied from the docs on how to document commands, usually because the commands docs can get bigger, but on this case we can go with the more normal style:

https://go.dev/doc/comment (the Commands section)

Reply:
ok no problem, I was curious but indeed makes sense for huge comments.

Reply:
I changed it for now since the comment is small, then we adapt accordingly in the future :+1: